.changeset/icy-dancers-brake.md

```
@@ -0,0 +1,11 @@
+---
+"@flowblade/source-duckdb": patch
+"@flowblade/source-kysely": patch
+"@flowblade/sqlduck": patch
+---
+
+Minimum logtape is now 2.2.2 in peer-deps
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)